src/components/Header.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Header from './Header';
+
+test('renders correctly', () => {
+  const Nav = renderer
+    .create(<Router><Header /></Router>)
+    .toJSON();
+  expect(Nav).toMatchSnapshot();
```

Comment:
- [ ] This test expectation fails to match the snapshot, kindly fix this to make sure the test passes.

Reply:
Thank you. Fixed